app/views/admin_incoming_message/bulk_destroy.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Confirm incoming message deletion</h1>
+
+<% @incoming_messages.each do |incoming_message| %>
+  <b><%= h(incoming_message.mail_from) %></b>
+  <%=_("at")%> <%=admin_value(incoming_message.sent_at)%>
+  <blockquote class="incoming-message">
+    <% if !incoming_message.cached_main_body_text_folded.nil? %>
+      <%= truncate(
+            incoming_message.cached_main_body_text_folded.
```

Comment:
I think this is causing https://github.com/mysociety/alaveteli/pull/3063/files#r54095767; I think it should be calling `#get_main_body_text_folded`, but it wouldn't surprise me if the the conditional was put there for performance reasons.